tests/behat/servercommunication.feature

```diff
@@ -15,16 +15,11 @@ Feature: Student can send answer and receive assessment from Aalto ASR
       | activity    | name               | intro               | course | idnumber  |
       | digitala    | Test digitala name | Test digitala intro | C1     | digitala1 |
     And I log in as "student1"
-    And I am on "Course 1" course homepage
+
+  Scenario: On assignment page a submit button is shown after recording
+    When I am on "Course 1" course homepage
     And I click on "Test digitala name" "link"
     And I click on "Assignment" "link"
     And I click on "record" "button"
-    Then I wait 2 seconds
```

Comment:
Here I had to delete one scenario with the waiting line, because it's not a feasible command. Requires further investigation.